1089/en.md

```diff
@@ -0,0 +1,109 @@
+# LOJ 1089 - Points in Segments (II)
+
+## Summary
+The problem involves a set of one-dimensional segments and a set of points. The objective is to count, for each point, how many segments include that point within their range. 
+
+Note: **AA** is a valid segment.
+
+## Solution
+First let's simplify the problem a bit. Suppose there is no such segment like: **AA** 
+
+Let's think of each segment **AB** as of two points where **A** is the opening of the segment and **B** the ending of it. 
+
+Consider the set **S** of all opening and ending points of the segments sorted in increasing order. Iterating over the set from the begining, whenever we find an opening point **S[i]**, we increment the **counter** (starting with 0) by 1, indicating for any point **p** such that **S[i] <= p <= S[i+1]**,  if **S[i+1]** is an ending point, otherwise **S[i] <= p < S[i+1]** will be contained by that many of segments. If we find an ending point **S[i]**, we know for sure we had the opening point of that segment before, so it's time to close the segment, so we decrement the **counter** by 1, now indicating for any point **p** such that **S[i] < p <= S[i+1]**,  if **S[i+1]** is an ending point, otherwise **S[i] < p < S[i+1]** will be contained by that many of segments.
```

Comment:
Let's take a step back and see if we can simplify this paragraph. I understand what you are trying to explain here, but I feel there are too many complicated sentences jumbled together. Let's explain it simply.
While doing so, let's also deviate away from any implementation details, for example, not bother explaining how you may write this in code.

Reply:
Simplified!